src/Shared/Debugger/EnumerableStringValuesDebugView.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNetCore.Shared;
+
+internal sealed class EnumerableStringValuesDebugView
```

Comment:
Nit:

```suggestion
internal sealed class StringValuesDictionaryDebugView
```

Or maybe

```suggestion
internal sealed class EnumerableKeyStringValuesPairDebugView
```

Reply:
I'll rename it to `StringValuesDictionaryDebugView`. It's not technically correct because it works with an enumerable, but in reality all the types it is used on are either dictionaries or dictionary-like.